packages/cfd/src/Components/__tests__/cfd-real-account-display.spec.js

```diff
@@ -0,0 +1,627 @@
+                        services: {
+                            idv: {
+                                documents_supported: {},
+                                has_visual_sample: 0,
+                                is_country_supported: 0,
+                            },
+                            onfido: {
+                                documents_supported: {
+                                    driving_licence: {
+                                        display_name: 'Driving Licence',
+                                    },
+                                    national_identity_card: {
+                                        display_name: 'National Identity Card',
+                                    },
+                                    passport: {
+                                        display_name: 'Passport',
+                                    },
+                                },
+                                is_country_supported: 1,
+                            },
+                        },
+                    },
+                    phone_idd: '62',
+                    text: 'Indonesia',
+                    tin_format: ['^\\d{15}$'],
+                    value: 'id',
+                },
+                {
+                    identity: {
+                        services: {
+                            idv: {
+                                documents_supported: {},
+                                has_visual_sample: 0,
+                                is_country_supported: 0,
+                            },
+                            onfido: {
+                                documents_supported: {},
+                                is_country_supported: 0,
+                            },
+                        },
+                    },
+                    phone_idd: '35818',
+                    text: 'Aland Islands',
+                    value: 'ax',
+                },
+            ],
+            standpoint: {
+                financial_company: 'svg',
+                gaming_company: 'svg',
+                iom: false,
+                malta: false,
+                maltainvest: false,
+                svg: true,
+            },
+            toggleAccountsDialog: jest.fn(),
+            toggleShouldShowRealAccountsList: jest.fn(),
+        };
+    });
+
+    it('should render Synthetic, Financial & Financial STP cards with enabled buttons on DMT5 when is_logged_in=true is_eu=false', () => {
+        render(<CFDRealAccountDisplay {...props} />);
+
+        const add_real_account_buttons = screen.getAllByRole('button', { name: /add real account/i });
+
+        expect(screen.getByTestId('dt_cfd_real_accounts_display')).toBeInTheDocument();
+        expect(screen.getByText('Synthetic')).toBeInTheDocument();
+        expect(screen.getByText('Financial')).toBeInTheDocument();
+        expect(screen.getByText('Financial STP')).toBeInTheDocument();
+        expect(screen.getAllByRole('table').length).toBe(3);
```

Comment:
Do we really need to check the number of tables? I think it's going to cause troubles since we might just want to update a table in the future and we will have to update the multiple test cases after that.

Reply:
good point 👍🏻 no need to additionally check the number of tables here since it's gonna be always equal to the number of account cards rendered